.pre-commit-config.yaml

```diff
@@ -20,21 +20,6 @@ default_language_version:
   # force all unspecified python hooks to run python3
   python: python3
 repos:
-  - repo: local
-    hooks:
-      - id: build
-        name: Check if image build is needed
-        entry: ./scripts/ci/local_ci_build.sh
-        language: system
-        always_run: true
-        pass_filenames: false
-      - id: check-apache-license
```

Comment:
Why did we move these? (I'm not too familiary with with a "repo" is in pre-commit terms.

Reply:
I moved those so that checking for image is done as late as possible - just before it is needed. Most pre-commit checks do not require breeze's image - only few of them (mypy/pylint/flake) are done using image to make sure they are run in the same environment. By moving "build" closer to the end, we can get those (much faster) pre-commit checks to be executed immediately and give feedback immediately - then somebody might decide to skip building image (and press Ctrl-C) seeing some other errors.

Reply:
I hope that explains - it's merely optimisation of user experience.